diff --git a/TVTComment/Model/SettingFileReaderWriter.cs b/TVTComment/Model/SettingFileReaderWriter.cs
index 20b5eed..9340a13 100644
--- a/TVTComment/Model/SettingFileReaderWriter.cs
+++ b/TVTComment/Model/SettingFileReaderWriter.cs
@@ -35,7 +35,8 @@ namespace TVTComment.Model
             jsonSerializerOptions = new JsonSerializerOptions()
             {
                 NumberHandling = JsonNumberHandling.AllowNamedFloatingPointLiterals |
-                                 JsonNumberHandling.AllowReadingFromString
+                                 JsonNumberHandling.AllowReadingFromString,
+                WriteIndented = true
             };
             jsonSerializerOptions.Converters.Add(new TimeSpanConverter());
             jsonSerializerOptions.Converters.Add(new JsonStringEnumConverter());
