diff --git a/TVTComment/Model/ChatCollectService/PastNichanChatCollectService.cs b/TVTComment/Model/ChatCollectService/PastNichanChatCollectService.cs
index 8ba4c05..8a0cf5f 100644
--- a/TVTComment/Model/ChatCollectService/PastNichanChatCollectService.cs
+++ b/TVTComment/Model/ChatCollectService/PastNichanChatCollectService.cs
@@ -8,6 +8,7 @@ using System.Runtime.ExceptionServices;
 using System.Text.RegularExpressions;
 using System.Threading;
 using System.Threading.Tasks;
+using Nichan;
 
 namespace TVTComment.Model.ChatCollectService
 {
@@ -151,40 +152,53 @@ namespace TVTComment.Model.ChatCollectService
         private static async Task<Nichan.Thread> GetThread(string server, string board, string thread, CancellationToken cancellationToken)
         {
             Nichan.Thread ret;
-            // まず2ch.scのdatから取得する
-            string datUrl = $"https://{server}.2ch.sc/{board}/dat/{thread}.dat";
-            string datResponse = null;
 
-            System.Diagnostics.Debug.WriteLine($"[PastNichanChatCollectService] HTTP Get {datUrl}");
-
-            try
-            {
-                datResponse = await httpClient.GetStringAsync(datUrl, cancellationToken).ConfigureAwait(false);
-            }
-            catch (HttpRequestException)
+            if (board != "livegalileo")
             {
-            }
+                // まず2ch.scのdatから取得する
+                string datUrl = $"http://{server}.2ch.sc/{board}/dat/{thread}.dat";
+                string datResponse = null;
 
-            if (datResponse != null)
-            {
-                ret = new Nichan.Thread() { Name = thread };
-                var datParser = new Nichan.DatParser();
-                datParser.Feed(datResponse);
+                System.Diagnostics.Debug.WriteLine($"[PastNichanChatCollectService] HTTP Get {datUrl}");
+
+                try
+                {
+                    datResponse = await httpClient.GetStringAsync(datUrl, cancellationToken).ConfigureAwait(false);
+                }
+                catch (HttpRequestException)
+                {
+                }
 
-                ret.Uri = new Uri(datUrl);
-                ret.Title = datParser.ThreadTitle;
-                while (true)
+                if (datResponse != null)
                 {
-                    Nichan.Res res = datParser.PopRes();
-                    if (res == null) break;
-                    ret.Res.Add(res);
+                    ret = new Nichan.Thread() { Name = thread };
+                    var datParser = new Nichan.DatParser();
+
+                    try
+                    {
+                        datParser.Feed(datResponse);
+                    }
+                    catch (DatParserException)
+                    {
+                        return default;
+                    }
+
+                    ret.Uri = new Uri(datUrl);
+                    ret.Title = datParser.ThreadTitle;
+                    while (true)
+                    {
+                        Nichan.Res res = datParser.PopRes();
+                        if (res == null) break;
+                        ret.Res.Add(res);
+                    }
+
+                    ret.ResCount = ret.Res.Count;
+                    return ret;
                 }
-                ret.ResCount = ret.Res.Count;
-                return ret;
             }
 
             // 2ch.scがダメだった場合、5ch.netのスクレイピング
             string gochanUrl = $"https://{server}.5ch.net/test/read.cgi/{board}/{thread}/";
             string response;
 
@@ -237,7 +251,13 @@ namespace TVTComment.Model.ChatCollectService
             ExceptionDispatchInfo.Capture(e).Throw(); // スタックトレースを保って再スロー
         }
 
-        private static readonly HttpClient httpClient = new HttpClient();
+        private static readonly HttpClient httpClient = new HttpClient
+        {
+            DefaultRequestHeaders =
+            {
+                {"User-Agent", "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.0.0 Safari/537.36"}
+            }
+        };
         private readonly NichanUtils.INichanThreadSelector threadSelector;
         private readonly TimeSpan threadSelectionUpdateInterval;
         private readonly List<Nichan.Thread> threadList = new List<Nichan.Thread>();
diff --git a/TVTComment/Model/Nichan/PastThreadLister.cs b/TVTComment/Model/Nichan/PastThreadLister.cs
index 6298e00..53cb228 100644
--- a/TVTComment/Model/Nichan/PastThreadLister.cs
+++ b/TVTComment/Model/Nichan/PastThreadLister.cs
@@ -1,8 +1,11 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.IO;
 using System.Linq;
+using System.Linq.Expressions;
 using System.Net.Http;
+using System.Net.Http.Headers;
 using System.Runtime.CompilerServices;
 using System.Text;
 using System.Text.RegularExpressions;
@@ -75,7 +78,13 @@ namespace Nichan
             }
         }
 
-        private static readonly HttpClient httpClient = new HttpClient();
+        private static readonly HttpClient httpClient = new HttpClient
+        {
+            DefaultRequestHeaders =
+            {
+                {"User-Agent", "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.0.0 Safari/537.36"}
+            }
+        };
         private readonly ArchivedThreadListRetriever threadListRetriever;
         private readonly TimeSpan backTime;
         /// <summary>
@@ -89,7 +98,8 @@ namespace Nichan
             var reses = new List<Res>();
 
             Match match = reThreadUrl.Match(threadUrl);
-            if (match.Success)
+            // なんGには 2ch.sc がない
+            if (match.Success && match.Groups["board"].Value != "livegalileo")
             {
                 // まず2ch.scのdatから取得する
                 (string server, string board, string threadId) = (match.Groups["server"].Value, match.Groups["board"].Value, match.Groups["thread"].Value);
@@ -109,7 +119,15 @@ namespace Nichan
                 if (datResponse != null)
                 {
                     var datParser = new DatParser();
-                    datParser.Feed(datResponse);
+
+                    try
+                    {
+                        datParser.Feed(datResponse);
+                    }
+                    catch (DatParserException)
+                    {
+                        return (default, default);
+                    }
 
                     while (true)
                     {
@@ -135,12 +153,9 @@ namespace Nichan
                 {
                     response = await httpClient.GetStringAsync(threadUrl, cancellationToken).ConfigureAwait(false);
                 }
-                catch (HttpRequestException e)
+                catch (HttpRequestException)
                 {
-                    if (e.StatusCode == null)
-                        throw new NetworkException(threadUrl, null, e);
-                    else
-                        throw new HttpErrorResponseException((int)e.StatusCode.Value, null, threadUrl, null, e);
+                    return (default, default);
                 }
 
                 using var textReader = new StringReader(response);
@@ -150,10 +165,10 @@ namespace Nichan
 
             var firstRes = reses[0];
             var lastRes = reses.FindLast(x => x.Number <= 1000 && x.Date != null);
-            var start = firstRes.Date == null ? null : (DateTimeOffset?)new DateTimeOffset(firstRes.Date.Value, TimeSpan.FromHours(9));
-            var end = lastRes.Date == null ? null : (DateTimeOffset?)new DateTimeOffset(lastRes.Date.Value, TimeSpan.FromHours(9));
+            var start = firstRes.Date == null ? default : new DateTimeOffset(firstRes.Date.Value, TimeSpan.FromHours(9));
+            var end = lastRes.Date == null ? default : new DateTimeOffset(lastRes.Date.Value, TimeSpan.FromHours(9));
 
-            return (start.Value, end.Value);
+            return (start, end);
         }
     }
 }
