diff --git a/TVTComment/Model/IPCModule.cs b/TVTComment/Model/IPCModule.cs
index c66ebf4..5630d5d 100644
--- a/TVTComment/Model/IPCModule.cs
+++ b/TVTComment/Model/IPCModule.cs
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.Diagnostics;
 using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
@@ -100,6 +101,7 @@ namespace TVTComment.Model
             }
             catch (Exception e)
             {
+                Debug.WriteLine(e.ToString());
                 //不明な例外が投げられた時は以降操作できない
                 await Dispose(IPCModuleDisposeReason.UnexpectedError, e);
                 throw;
@@ -145,6 +147,7 @@ namespace TVTComment.Model
             }
             catch (Exception e)
             {
+                Debug.WriteLine(e.ToString());
                 await Dispose(IPCModuleDisposeReason.UnexpectedError, e).ConfigureAwait(false);
                 throw;
             }
@@ -190,6 +193,7 @@ namespace TVTComment.Model
             }
             catch (Exception e)
             {
+                Debug.WriteLine(e.ToString());
                 await Dispose(IPCModuleDisposeReason.UnexpectedError, e).ConfigureAwait(false);
             }
         }
