diff --git a/TVTComment/Views/ShellContents/BasicSettingControl.xaml b/TVTComment/Views/ShellContents/BasicSettingControl.xaml
index ad9c642..cdaa119 100644
--- a/TVTComment/Views/ShellContents/BasicSettingControl.xaml
+++ b/TVTComment/Views/ShellContents/BasicSettingControl.xaml
@@ -89,6 +89,8 @@
         </TextBlock>
         <TextBlock>
             © 2017 silane / © 2020-2021 noriokun4649
+            <LineBreak />
+            Patched by @SlashNephy &lt;spica@starry.blue&gt;
         </TextBlock>
     </StackPanel>
 </UserControl>
