diff --git a/TVTComment/Properties/PublishProfiles/ReleaseProfile.pubxml b/TVTComment/Properties/PublishProfiles/ReleaseProfile.pubxml
index 65d7e3a..5fddc51 100644
--- a/TVTComment/Properties/PublishProfiles/ReleaseProfile.pubxml
+++ b/TVTComment/Properties/PublishProfiles/ReleaseProfile.pubxml
@@ -9,9 +9,9 @@ https://go.microsoft.com/fwlink/?LinkID=208121.
     <PublishDir>bin\Release\net5.0-windows\publish\</PublishDir>
     <PublishProtocol>FileSystem</PublishProtocol>
     <TargetFramework>net5.0-windows</TargetFramework>
-    <SelfContained>true</SelfContained>
+    <SelfContained>false</SelfContained>
     <RuntimeIdentifier>win-x86</RuntimeIdentifier>
-    <PublishSingleFile>True</PublishSingleFile>
+    <PublishSingleFile>False</PublishSingleFile>
     <PublishReadyToRun>False</PublishReadyToRun>
     <PublishTrimmed>False</PublishTrimmed>
     <IncludeAllContentForSelfExtract>true</IncludeAllContentForSelfExtract>
