diff --git a/TVTComment/Model/ChatCollectService/TwitterLiveChatCollectService.cs b/TVTComment/Model/ChatCollectService/TwitterLiveChatCollectService.cs
index 9f761a8..66fdd6b 100644
--- a/TVTComment/Model/ChatCollectService/TwitterLiveChatCollectService.cs
+++ b/TVTComment/Model/ChatCollectService/TwitterLiveChatCollectService.cs
@@ -1,5 +1,5 @@
-﻿using CoreTweet;
-using CoreTweet.Streaming;
+﻿using CoreTweet.V1;
+using CoreTweet.V1.Streaming;
 using ObservableUtils;
 using System;
 using System.Collections.Concurrent;
@@ -94,11 +94,11 @@ namespace TVTComment.Model.ChatCollectService
         {
             await Task.Run(() =>
             {
-                foreach (var status in Twitter.Token.Streaming.Filter(track: searchWord)
-                            .OfType<StatusMessage>()
-                            .Where(x => !x.Status.Text.StartsWith("RT"))
-                            .Where(x => x.Status.Language is null or "und" || x.Status.Language.StartsWith("ja"))
-                            .Select(x => x.Status))
+                foreach (var status in Twitter.Token.V1.Streaming.Filter(track: searchWord)
+                             .OfType<StatusMessage>()
+                             .Where(x => !x.Status.Text.StartsWith("RT"))
+                             .Where(x => x.Status.Language is null or "und" || x.Status.Language.StartsWith("ja"))
+                             .Select(x => x.Status))
                 {
                     if (cancel.IsCancellationRequested || !SearchWord.Value.Equals(searchWord))
                         break;
@@ -153,7 +153,7 @@ namespace TVTComment.Model.ChatCollectService
             try
             {
                 var suffix = "\n" + (postObject as ChatPostObject)?.SuffixText ?? "";
-                await Twitter.Token.Statuses.UpdateAsync(postObject.Text + suffix);
+                await Twitter.Token.V1.Statuses.UpdateAsync(postObject.Text + suffix);
             }
             catch
             {
diff --git a/TVTComment/Model/ChatCollectService/TwitterLiveV2ChatCollectService.cs b/TVTComment/Model/ChatCollectService/TwitterLiveV2ChatCollectService.cs
index b720c52..5bbe98d 100644
--- a/TVTComment/Model/ChatCollectService/TwitterLiveV2ChatCollectService.cs
+++ b/TVTComment/Model/ChatCollectService/TwitterLiveV2ChatCollectService.cs
@@ -154,7 +154,7 @@ namespace TVTComment.Model.ChatCollectService
             try
             {
                 var suffix = "\n" + (postObject as ChatPostObject)?.SuffixText ?? "";
-                await Twitter.Token.Statuses.UpdateAsync(postObject.Text + suffix);
+                await Twitter.Token.V1.Statuses.UpdateAsync(postObject.Text + suffix);
             }
             catch
             {
diff --git a/TVTComment/Model/ChatService/TwitterChatService.cs b/TVTComment/Model/ChatService/TwitterChatService.cs
index f04f14e..6af258c 100644
--- a/TVTComment/Model/ChatService/TwitterChatService.cs
+++ b/TVTComment/Model/ChatService/TwitterChatService.cs
@@ -112,7 +112,7 @@ namespace TVTComment.Model.ChatService
             var twitterAuthentication = new TwitterAuthentication(apiKey, apiSecret, apiAccessToken, apiAccessSecret, bearerToken);
             twitterAuthentication.Login();
             var tokens = twitterAuthentication.Token;
-            var userResponse = await tokens.Account.VerifyCredentialsAsync().ConfigureAwait(false);
+            var userResponse = await tokens.V1.Account.VerifyCredentialsAsync().ConfigureAwait(false);
             IsLoggedin = true;
             UserName = userResponse.Name;
             settings.ApiKey = apiKey;
@@ -152,7 +152,7 @@ namespace TVTComment.Model.ChatService
                 throw new TwiiterAuthException("まずは「OAuth画面を開く」を押してPINコードを表示させてください");
             await twitterAuthentication.Login(pin).ConfigureAwait(false);
             var tokens = twitterAuthentication.Token;
-            var userResponse = await tokens.Account.VerifyCredentialsAsync().ConfigureAwait(false);
+            var userResponse = await tokens.V1.Account.VerifyCredentialsAsync().ConfigureAwait(false);
             IsLoggedin = true;
             UserName = userResponse.Name;
             twitterSession.Value = twitterAuthentication;
diff --git a/TVTComment/Model/TwitterUtils/ChatTwitterStatusToChat.cs b/TVTComment/Model/TwitterUtils/ChatTwitterStatusToChat.cs
index d8e5c1f..4fd2437 100644
--- a/TVTComment/Model/TwitterUtils/ChatTwitterStatusToChat.cs
+++ b/TVTComment/Model/TwitterUtils/ChatTwitterStatusToChat.cs
@@ -1,4 +1,4 @@
-﻿using CoreTweet;
+﻿using CoreTweet.V1;
 using CoreTweet.V2;
 using System.Drawing;
 
